deploy/dynamo/sdk/src/dynamo/sdk/cli/serve.py

```diff
@@ -71,21 +71,6 @@ def cli():
         type=click.Path(exists=True),
         help="Path to YAML config file for service configuration",
     )
-    @click.option(
```

Comment:
Will we want to bring back customizability of port and host? I can see the usefulness of it especially to resolve port conflicts